easybuild/easyconfigs/f/foss/foss-2019a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'Toolchain'
+
+name = 'foss'
+version = '2019a'
+
+homepage = '(none)'
```

Comment:
Could link to https://easybuild.readthedocs.io/en/develop/Common-toolchains.html#foss-toolchain here